app/models/services/service_key.rb

```diff
@@ -0,0 +1,93 @@
+module VCAP::CloudController
+  class ServiceKey < Sequel::Model
+    class InvalidAppAndServiceRelation < StandardError; end
+
+    many_to_one :service_instance
+
+    export_attributes :name, :service_instance_guid, :credentials, :syslog_drain_url
```

Comment:
`syslog_drain_url` is not applicable to service_keys (it only matters for bindings with apps). Please remove all fields and logic regarding syslog_drain_url